cfme/tests/cli/test_appliance_console.py

```diff
@@ -1291,7 +1290,10 @@ def test_appliance_console_extend_storage_negative():
             2.
             3. Check there is no extra disk present.
     """
-    pass
+    command_set = ("ap", RETURN, "11")
+    result = appliance.appliance_console.run_commands(command_set, timeout=30, output=True)
+    assert [i for i in result[-1].split("\n") if "/dev/vd" not in i.strip()], (
+        "extra disks are not presents.")
```

Comment:
:floppy_disk: :no_entry_sign: :gift: small typo
```suggestion
        "extra disks are not present.")
```

Reply:
Done